src/main/steps/urls.ts

```diff
@@ -281,6 +281,7 @@ export const C100_REASONABLE_ADJUSTMENTS_DOCUMENT_INFORMATION: PageLink = `${C10
 export const C100_REASONABLE_ADJUSTMENTS_COMMUNICATION_HELP: PageLink = `${C100_URL}/reasonable-adjustments/disability-requirements/communication-help`;
 export const C100_REASONABLE_ADJUSTMENTS_FEEL_COMFORTABLE: PageLink = `${C100_URL}/reasonable-adjustments/disability-requirements/feel-comfortable`;
 export const C100_REASONABLE_ADJUSTMENTS_TRAVELLING_COURT: PageLink = `${C100_URL}/reasonable-adjustments/disability-requirements/travelling-court`;
+export const C100_COURT_PERMISSION: PageLink = `${C100_URL}/permission`;
```

Comment:
@ShakiraKP this is part of screen questions, please append /screen-questions as base then /permission.
c100-rebuild/screening-questions/permission

Reply:
Okay no problem will adjust, I didnt know where it belonged as the link in the prototype is very basic but I will change now